dev_guide/builds.adoc

```diff
@@ -5,19 +5,40 @@
 :icons:
 :experimental:
 :toc: macro
+:toclevels: 3
 :toc-title:
 :prewrap!:
 
 toc::[]
 
 == Overview
-A link:../architecture/core_concepts/builds_and_image_streams.html#builds[build] is a process of creating
-runnable images to be used on OpenShift. There are three build strategies:
+
+A link:../architecture/core_concepts/builds_and_image_streams.html#builds[build]
+is a process of creating runnable images to be used on OpenShift.
+
+Build configurations are characterized by strategy and sources. The strategy
+determines the aforementioned process, while the sources provide its input.
+Every build configuration have a single strategy and at least one source.
```

Comment:
I had already updated that paragraph removing the last sentence and applying your suggestion to the first: "... characterized by _a_ strategy and _one or more_ sources".